swagger.json

```diff
@@ -1821,6 +1821,57 @@
             }
           }
         },
+        "/fleet/vehicles/{vehicle_id}/stats": {
+            "parameters": [
+                {
+                  "$ref": "#/parameters/accessTokenParam"
+                },
+                {
+                  "name": "vehicle_id",
+                  "in": "path",
+                  "required": true,
+                  "description": "ID of the vehicle.",
+                  "type": "integer",
+                  "format": "int64"
+                },
+                {
+                  "name": "startMs",
+                  "description": "Time in Unix epoch milliseconds for the start of the query (cannot exceed 1 hour)",
+                  "required": true,
+                  "in": "query",
+                  "type": "integer",
+                  "format": "int64"
+                },
+                {
+                  "name": "endMs",
+                  "description": "Time in Unix epoch milliseconds for the end of the query (cannot exceed 1 hour)",
+                  "required": true,
+                  "in": "query",
+                  "type": "integer",
+                  "format": "int64"
+                }
+              ],
+              "get": {
+                  "tags": ["Fleet"],
+                  "summary": "/fleet/vehicles/{vehicle_id:[0-9]+}/stats",
+                  "description": "Fetch engine state and digital aux data for a given vehicle between a start/end time. The maximum query duration is one hour.",
```

Comment:
Left comment within [mini dev spec](https://paper.dropbox.com/doc/Mini-Dev-Spec-Pike-Electric-API-Updates--AU8GqZ4wvVh5~ma2YfbYZ5u5Ag-wzPIvSmJf8MVofdtJXml6). I'm curious to know if this approach is scalable, and if it would be possible to instead have the developer pass in parameters to request certain object stats.

Reply:
Added a `filter` argument where caller can request particular stats.